care/facility/models/patient.py

```diff
@@ -436,6 +437,14 @@ class TestTypeEnum(enum.Enum):
 
     objects = BaseManager()
 
+    @property
+    def is_expired(self) -> bool:
+        return (
+            self.last_consultation.new_discharge_reason
+            == NewDischargeReasonEnum.EXPIRED.value
+            or self.last_consultation.discharge_reason == "EXP"
+        )
```

Comment:
@khavinshankar we could instead use the `death_datetime` field from the patient registration table itself which was introduced in #1966 to identify whether the patient is dead or not right?

<img width="1488" alt="image" src="https://github.com/coronasafe/care/assets/25143503/c4125cd7-a5d8-4a4e-b392-76eef7f52a24">